src/globalComponents/typesAndClasses/Replay.js

```diff
@@ -1,24 +1,31 @@
 //@flow strict
+import type { Map } from './Map.js';
 
 class Replay {
-	tankOneName: string;
-	tankTwoName: string;
+	tankOneName: ?string;
+	tankTwoName: ?string;
 	playerOneName: string;
 	playerTwoName: string;
 	winner: -1|0|1|2;
 	prizeMoney: number;
 	eloExchanged: number;
 	replayId: string;
+	map: Map;
+	tankTeamOneNames: Array<?string>;
+	tankTeamTwoNames: Array<?string>;
 
 	constructor(
-		tankOneName: string,
-		tankTwoName: string,
+		tankOneName: ?string,
+		tankTwoName: ?string,
 		playerOneName: string,
 		playerTwoName: string,
 		winner: -1|0|1|2,
 		prizeMoney: number,
 		eloExchanged: number,
 		replayId: string,
+		map: Map,
```

Comment:
Yeah, we should probably be consistant with this and either use Map everywhere or use ArenaType everywhere.